static/js/popovers.js

```diff
@@ -136,6 +136,20 @@ exports.hide_actions_popover = function () {
     }
 };
 
+exports.topic_ops = {
+    mute: function (stream, topic) {
+        popovers.hide_topic_sidebar_popover();
+        muting.mute_topic(stream, topic, true);
+        muting_ui.persist_and_rerender();
+        muting_ui.mute_notification(stream, topic);
```

Comment:
Consider a name change here that is more verb-y, like `muting_ui.notify_with_undo_option(stream, topic)`.